File: android/StarWarsJavaRX/app/src/main/java/br/com/nglauber/starwarsjavarx/model/api/StarWarsApiDef.java
package br.com.nglauber.starwarsjavarx.model.api;

import retrofit2.http.GET;
import retrofit2.http.Path;
import rx.Observable;

public interface StarWarsApiDef {
    @GET("films")
    Observable<FilmResult> listMovies();

    @GET("people/{personId}")
    Observable<Person> loadPerson(@Path("personId")String personId);
}

File: android/StarWarsKoltinRX/app/src/main/java/br/com/nglauber/starwarskoltinrx/model/api/StarWarsApiDef.kt
package br.com.nglauber.starwarskoltinrx.model.api

import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Path

interface StarWarsApiDef {
    @GET("films")
    fun listMovies() : Observable<FilmResult>

    @GET("people/{personId}")
    fun loadPerson(@Path("personId") personId : String) : Observable<Person?>
}